### PR TITLE
Update configure.md

### PR DESCRIPTION
**Pull request by rnovikovP**
Add Clafirication about "what happened if I'll enable and then disable feature "

### CODE REVIEW

File: docs/how-to/configure.md [hunk 1]
```diff
@@ -130,16 +137,22 @@ Enables DBaaS features on this server.
...
 
+If you'll disable the already enabled Integrated Alerting feature you will stop receiving the notifications and will not be able to access the UI of this feature 
+
 ### Microsoft Azure Monitoring
 
 > <b style="color:goldenrod">Caution</b> This is a technical preview feature.
 
 Activates Microsoft Azure monitoring.
 
+If you'll disable the already enabled Microsoft Azure monitoring feature you will not be able to do discovery and add Azure instances using the native  UI from PMM. No date related to already added Azure instances will not be deleted and already added instances will be monitored
```
Comment (BupycHuk) [inline docs/how-to/configure.md:154]:
```suggestion
If you'll disable the already enabled Microsoft Azure monitoring feature you will not be able to do discovery and add Azure instances using the native  UI from PMM. Data related to already added Azure instances will not be deleted and already added instances will be monitored 
```

File: docs/how-to/configure.md [hunk 2]
```diff
@@ -85,11 +85,16 @@ As well as via the *PMM Settings* page, you can also disable telemetry with the
 > - If the Security Threat Tool is enabled in PMM Settings, Telemetry is automatically enabled.
 >
 > - Telemetry is sent immediately; the 24-hour grace period is not honored.
+> 
+
+If you'll disable the already enabled Telemetry feature PMM will stop sending data and all feature where you can get some data from Percona will be not available to be enabled
```
Comment (PaulJacobs-percona) [inline docs/how-to/configure.md:90]:
```suggestion
>
> - Disabling telemetry prevents you from using features that require data from us.
```

File: docs/how-to/configure.md [hunk 3]
```diff
@@ -85,11 +85,16 @@ As well as via the *PMM Settings* page, you can also disable telemetry with the
...
 > - Telemetry is sent immediately; the 24-hour grace period is not honored.
+> 
+
+If you'll disable the already enabled Telemetry feature PMM will stop sending data and all feature where you can get some data from Percona will be not available to be enabled 
 
 ### Check for updates
 
 When active, PMM will automatically check for updates and put a notification in the home page *Updates* dashboard if any are available.
 
+If you'll disable the already enabled Check for Updates feature you will not be longer informed about new versions of PMM and might miss important features we'll ad in future
```
Comment (PaulJacobs-percona) [inline docs/how-to/configure.md:96]:
```suggestion
If you turn this off you will not know about important updates and features without watching our forum and release notes pages.
```

Comment (PaulJacobs-percona) [reply docs/how-to/configure.md:96]: But I'd rather omit this entirely, as our readers/users are smart and can figure it out for themselves.

File: docs/how-to/configure.md [hunk 4]
```diff
@@ -120,6 +125,8 @@ Checks use the *Standard* interval by default. To change a check's interval:
 
 (Read more at [Security Threat Tool](../using/platform/security-threat-tool.md).)
 
+If you'll disable the already enabled Security Threat Tool  feature you will not be able to see the Security alerts and Security Checks will not be executed on the system
```
Comment (PaulJacobs-percona) [inline docs/how-to/configure.md:128]: As above, we are just stating the obvious. We are in danger of insulting the intelligence of our readers with such words.

File: docs/how-to/configure.md [hunk 5]
```diff
@@ -130,16 +137,22 @@ Enables DBaaS features on this server.
 
 > <b style="color:goldenrod">Caution</b> DBaaS functionality is a technical preview that must be turned on with a server feature flag. See [DBaaS](../setting-up/server/dbaas.md).
 
+If you'll disable the already enabled DBaaS feature you will not lose any of your K8s data or DB cluster. The only effect this will have is the DBaaS menu will disappear from the PMM
```
Comment (PaulJacobs-percona) [inline docs/how-to/configure.md:140]:
```suggestion
This setting only shows and hides DBaaS features. Your Kubernetes and DB clusters are unaffected.
```

File: docs/how-to/configure.md [hunk 6]
```diff
@@ -130,16 +137,22 @@ Enables DBaaS features on this server.
 
 > <b style="color:goldenrod">Caution</b> DBaaS functionality is a technical preview that must be turned on with a server feature flag. See [DBaaS](../setting-up/server/dbaas.md).
 
+If you'll disable the already enabled DBaaS feature you will not lose any of your K8s data or DB cluster. The only effect this will have is the DBaaS menu will disappear from the PMM 
+
 ### Integrated Alerting
 
 Enables [Integrated Alerting](../using/alerting.md) and reveals the [Communication](#communication) tab.
 
+If you'll disable the already enabled Integrated Alerting feature you will stop receiving the notifications and will not be able to access the UI of this feature
```
Comment (PaulJacobs-percona) [inline docs/how-to/configure.md:146]:
```suggestion
Disabling this feature hides the related Integrated Alerting user interface and stops any alert notifications.
```

File: docs/how-to/configure.md (hunk 2 above)
Comment (PaulJacobs-percona) [reply docs/how-to/configure.md:90]: What features specifically are not available to the user when they disable telemetry? I have reworded an already vague sentence but it should be clearer.

**Review by PaulJacobs-percona — CHANGES_REQUESTED** [review]
These texts seem to be simply stating the negative of an already obvious thing. "When on, features works." You've added "When off, it doesn't." Is it really necessary?

**Pull request closed without merging**